README.md

```diff
@@ -2,15 +2,21 @@
 This repo aims to collect OSS made in Sudan or by Sudanese.
 Please feel free to submit a PR with your OSS (Package, App ...etc.), or OSS from someone you know.
 
-# Template:
+> For Contribute read [Contributing Guide](contributing.md) please
 
-- [Open Source Project Name](https://github.com/obiefy/api-response) - Small Description about the Open Source Project.
+## Plugins
```

Comment:
Please rename Plugins to packages. Or Add Plugins and Packages as separated categories.

Reply:
Done :v: